acceptance.bats

```diff
@@ -214,6 +214,12 @@ resetCacheFolder() {
   [ "`ls cache/ | wc -l`" -eq 0 ]
 }
 
+@test "Cache not only original schema but also refs to other schemas" {
+  run bin/kubeconform -schema-location './fixtures/registry/{{ .ResourceKind }}{{ .KindSuffix }}-with-ref.json' fixtures/test_crd.yaml
+  [ "$status" -eq 0 ]
+  [ "`ls cache/ | wc -l`" -eq 1 ]
```

Comment:
Actual value is 0 because no refs got cached

Reply:
I think the better test would be

1. Run kubeconform with internet access
2. Disable network
3. Run kubeconform again and check that status hasn't changed